CYD-Klipper/src/core/device/ESP32-CROWPANEL-70.cpp

```diff
@@ -0,0 +1,297 @@
+            
+            cfg.pin_d5  = GPIO_NUM_9;  // G0
+            cfg.pin_d6  = GPIO_NUM_46; // G1
+            cfg.pin_d7  = GPIO_NUM_3;  // G2
+            cfg.pin_d8  = GPIO_NUM_8;  // G3
+            cfg.pin_d9  = GPIO_NUM_16; // G4
+            cfg.pin_d10 = GPIO_NUM_1;  // G5
+            
+            cfg.pin_d11 = GPIO_NUM_14; // R0
+            cfg.pin_d12 = GPIO_NUM_21; // R1
+            cfg.pin_d13 = GPIO_NUM_47; // R2
+            cfg.pin_d14 = GPIO_NUM_48; // R3
+            cfg.pin_d15 = GPIO_NUM_45; // R4
+    
+            cfg.pin_henable = GPIO_NUM_41;
+            cfg.pin_vsync   = GPIO_NUM_40;
+            cfg.pin_hsync   = GPIO_NUM_39;
+            cfg.pin_pclk    = GPIO_NUM_0;
+            cfg.freq_write  = 15000000;
+    
+            cfg.hsync_polarity    = 0;
+            cfg.hsync_front_porch = 40;
+            cfg.hsync_pulse_width = 48;
+            cfg.hsync_back_porch  = 40;
+            
+            cfg.vsync_polarity    = 0;
+            cfg.vsync_front_porch = 1;
+            cfg.vsync_pulse_width = 31;
+            cfg.vsync_back_porch  = 13;
+    
+            cfg.pclk_active_neg   = 1;
+            cfg.de_idle_high      = 0;
+            cfg.pclk_idle_high    = 0;
+    
+            _bus_instance.config(cfg);
+        }
+                {
+            auto cfg = _panel_instance.config();
+            cfg.memory_width  = 800;
+            cfg.memory_height = 480;
+            cfg.panel_width  = 800;
+            cfg.panel_height = 480;
+            cfg.offset_x = 0;
+            cfg.offset_y = 0;
+            _panel_instance.config(cfg);
+        }
+        _panel_instance.setBus(&_bus_instance);
+        setPanel(&_panel_instance);
+        }
+    }
+bool touch_touched()
+{
+  ts.read();
+  if (ts.isTouched)
+  {
+    touch_last_x = map(ts.points[0].x, TOUCH_MAP_X1, TOUCH_MAP_X2, 0, tft.width() - 1);
+    touch_last_y = map(ts.points[0].y, TOUCH_MAP_Y1, TOUCH_MAP_Y2, 0, tft.height() - 1);
+    return true;
+  }
+  else
+  {
+    return false;
+  }
+}
+
+bool touch_released()
+{
+  return true;
+}
+
+// screen driver
+
+void screen_setBrightness(unsigned char brightness)
+{
+    ledcWrite(1, brightness); /* Screen brightness can be modified by adjusting this parameter. (0-255) */  
+}
+
+void set_invert_display()
+{
+    // TODO
+}
+
+void screen_lv_flush(lv_disp_drv_t *disp, const lv_area_t *area, lv_color_t *color_p)
+{
+    uint32_t w = (area->x2 - area->x1 + 1);
+    uint32_t h = (area->y2 - area->y1 + 1);
+  
+    tft.pushImageDMA(area->x1, area->y1, w, h,(lgfx::rgb565_t*)&color_p->full);//
+    lv_disp_flush_ready(disp);
+}
+
+void screen_lv_touchRead(lv_indev_drv_t *indev_driver, lv_indev_data_t *data)
+{
+    if (touch_has_signal())
+    {
+      if (touch_touched())
+      {
+        data->state = LV_INDEV_STATE_PR;
+  
+        /*Set the coordinates*/
+        data->point.x = touch_last_x;
+        data->point.y = touch_last_y;
+        //Serial.print( "Data x " );
```

Comment:
Would you mind removing commented out code? (I know the original driver has this too. If you feel like it, could you remove it there too?)